packages/jest-reporters/package.json

```diff
@@ -5,7 +5,9 @@
   "main": "build/index.js",
   "types": "build/index.d.ts",
   "dependencies": {
+    "@bcoe/v8-coverage": "^0.2.3",
```

Comment:
@bcoe this has an engine dependency on v10 - do you think that could be relaxed so people won't get engine warnings on node 8?

Reply:
I'm comfortable with doing this, since it's enforced in the consuming libraries; submit a patch if I don't get to it first?

Reply:
Happy to

Reply:
It actually wasn't this, but `v8-to-istanbul`.